apps/judicial-system/web/src/utils/validate.spec.ts

```
@@ -296,3 +298,15 @@ describe('Validate vehicle registration number', () => {
     },
   )
 })
+
+describe.only('isTrafficViolationStepValidIndictments', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)